tests/Unit/PointwiseFunctions/GeneralRelativity/Python/Test_GhBindings.py

```diff
@@ -0,0 +1,33 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import unittest
+
+import numpy as np
+import numpy.testing as npt
+
+from spectre.DataStructures import DataVector
+from spectre.DataStructures.Tensor import Scalar, tnsr
+from spectre.PointwiseFunctions.GeneralRelativity.GeneralizedHarmonic import *
```

Comment:
Instead of the `import *` let's do this:

```py
import spectre.PointwiseFunctions.GeneralRelativity.GeneralizedHarmonic as gh

# ...

gh.time_deriv_of_shift(...)
```

Then you don't have to worry about naming variables with trailing underscores like you did with `phi` and `pi` below.

Reply:
I've made all of the other changes requested, but I'm not sure how to occupy the '...' in each argument here.
Also, sorry if this isn't the place to request help.

Reply:
Below you already have

```py
        phi_ = tnsr.iaa[DataVector, 3](num_points=1, fill=1.0)
        pi_ = tnsr.aa[DataVector, 3](num_points=1, fill=1.0)
        time_deriv_of_shift(
            lapse,
            shift,
            inverse_spatial_metric,
            spacetime_unit_normal,
            phi_,
            pi_,
        )
```

if you make Nils' change, then this will become

```py
        phi = tnsr.iaa[DataVector, 3](num_points=1, fill=1.0)
        pi = tnsr.aa[DataVector, 3](num_points=1, fill=1.0)
        gh.time_deriv_of_shift(
            lapse,
            shift,
            inverse_spatial_metric,
            spacetime_unit_normal,
            phi,
            pi,
        )
```

Note the trailing `_` in pi and phi are gone and your function just has a `gh.` in front of it.